.github/workflows/build_wheel_manylinux2014.yml

```diff
@@ -37,16 +37,17 @@ jobs:
 
     steps:
       - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.9.1
+        uses: styfle/cancel-workflow-action@0.10.0
```

Comment:
The  `cancel-workflow-action` @0.11.0  was just released. Are we going to upgrade it to `0.11.0`?

Reply:
Thank you for reminding me of that.